src/codeu/chat/client/commandline/Chat.java

```diff
@@ -370,6 +373,29 @@ public void invoke(List<String> args) {
           }
         });
 
+    panel.register("c-remove", new Panel.Command(){
+      @Override
+      public void invoke(List<String> args){
+        final String name = args.size() > 0 ? args.get(0).trim() : "";
+        if(name.length() > 0){
```

Comment:
Prefer the following, which is more readable:

    if (name.isEmpty()) {
      System.out.println("ERROR: Wrong format");
      return;
    }

Generally, it's good to minimize the number of indentation levels, and put related code next as close as possible. So if we're effectively returning early with an error, express it this way. Also, this error can be more descriptive--maybe "no conversation specified"?